meteor/client/ui/RundownView.tsx

```diff
@@ -1291,6 +1288,11 @@ export const RundownView = translateWithTracker<IProps, IState, ITrackedProps>((
 })(
 	class RundownView extends MeteorReactComponent<Translated<IProps & ITrackedProps>, IState> {
 		private _hideNotificationsAfterMount: number | undefined
+		/** MiniShelf data */
+		private keyboardQueuedPiece: AdLibPieceUi | undefined = undefined
+		private keyboardQueuedPartInstanceId: PartInstanceId | undefined = undefined
+		private keyboardRequeue: boolean = false
+		private keyboardQueuePending: boolean = false
```

Comment:
Since `keyboardRequeue` and `keyboardQueuePending` are booleans it would be really nice if they were prefixed with "is"